eslintconfigs/module.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	'env': {
+		'browser': true
+	},
+	'parserOptions': {
+		'ecmaVersion': 5
```

Comment:
```suggestion
		'ecmaVersion': 5
```

I'm not confident this supports IE8-11 rules

Reply:
It just forbid to use es6 syntax e.g. `let`, `const`, etc